404.html

```diff
@@ -10,8 +7,8 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width">
   <title>Codezilla</title>
-  <link href="vector.css" rel="stylesheet" type="text/css" />
-  <link href="menu.css" rel="stylesheet" type="text/css" />
+  <link href="../vector.css" rel="stylesheet" type="text/css" />
+  <link href="../menu.css" rel="stylesheet" type="text/css" />
```

Comment:
Hey there - these links are fine, please revert them to the originals
The links to be changed are under the `<nav>` tag

For example, the navigation to home would be something like:
```diff
- <li class="item"> <a href="./home.html" class="link">
+ <li class="item"> <a href="./home" class="link">
    <svg> (...) </svg>
    Home
  </a></li>
```